src/harness/testcases/heartbeat_testcase.py

```diff
@@ -272,6 +272,136 @@ def test_WINNF_FT_S_HBT_4(self):
         self.assertEqual(resp['response']['responseCode'], 0)
 
   @winnforum_testcase
+  def test_WINNF_FT_S_HBT_5(self):
+    """SAS has requested CBSD to perform measurement through measReportConfig 
+    in the initial Grant response, or through a subsequent Heartbeat Response.
+
+    The response should be SUCCESS.
+    """
+
+    # Register the device
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    device_a['measCapability'] = ['EUTRA_CARRIER_RSSI_ALWAYS']
+    request = {'registrationRequest': [device_a]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
+    # Check registration response
+    self.assertEqual(response['response']['responseCode'], 0)
+    cbsd_id = response['cbsdId']
+    del request, response
+
+    # Request grant
+    grant_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_0['cbsdId'] = cbsd_id
+    request = {'grantRequest': [grant_0]}
+    # Check grant response
+    response = self._sas.Grant(request)['grantResponse'][0]
+    self.assertEqual(response['cbsdId'], cbsd_id)
+    self.assertTrue(response['grantId'])
+    self.assertEqual(response['response']['responseCode'], 0)
+    self.assertTrue(response['measReportConfig'])
+    grant_id = response['grantId']
+    grant_expire_time = datetime.strptime(response['grantExpireTime'],
+                                          '%Y-%m-%dT%H:%M:%SZ')
+    del request, response
+
+    # Heartbeat
+    request = {
```

Comment:
I think the test spec is wrong and that we need to include the measReport field here.